plugins/amd-kria/fu-amd-kria-device.c

```diff
@@ -132,17 +131,14 @@ fu_amd_kria_device_probe(FuDevice *device, GError **error)
 	 * Fetch the OF_FULLNAME udev property and look for the I2C address in it
 	 * sample format: OF_FULLNAME=/axi/i2c@ff030000/eeprom@50
 	 */
-	prop_of_fullname =
-	    fu_udev_device_read_property(FU_UDEV_DEVICE(device), "OF_FULLNAME", error);
-	if (prop_of_fullname == NULL)
-		return FALSE;
-	of_name = fu_strsplit(prop_of_fullname, strlen(prop_of_fullname), "@", -1);
+	tmp = fu_device_get_physical_id(device);
+	of_name = fu_strsplit(tmp, strlen(tmp), "@", -1);
```

Comment:
I think this is fragile; the physical ID isn't always going to be the OF_FULLNAME -- I think if we do want to support it for every UDev device we need to put it as a new property -- and that seems like a huge waste of a pointer for thousands of devices that aren't going to have one.